src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
The fields `loadCapacity` and `numberOfAxles` should have access modifiers to encapsulate the data. Consider declaring them as private to prevent direct access from outside the class.